cli/relevance_tests/test_precision.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+import pytest
+
+from . import get_pipeline_elastic_client, works_index, works_query
+
+client = get_pipeline_elastic_client()
+
+test_cases = [
+    pytest.param("horse", "pgwnkf2h", id="horse"),
+    pytest.param("cow", "wm8wy47d", id="cow"),
+]
```

Comment:
Aware that this is not a proper implementation but it's very unclear what these params are - I know this is partly pytest's fault, might be clearer to define the test cases inline in the `@pytest.mark.parametrize`?